app/src/components/features/rules/RuleBuilder/Header/ActionButtons/ShareRuleButton/index.jsx

```diff
@@ -16,6 +17,7 @@ const ShareRuleButton = ({ isRuleEditorModal }) => {
   const currentlySelectedRuleData = useSelector(getCurrentlySelectedRuleData);
 
   const shareRuleClickHandler = () => {
+    trackShareButtonClicked(1, "rule_editor");
```

Comment:
In `trackShareButtonClicked` function keep `source` param first and `selection_count` as second param, and give a default value to `selection_count = 1`, so no need to pass `selection_count` when there is only one rule